docs/README.md

```diff
@@ -1,2 +1,294 @@
+
+To fund your harmony account use the [harmony cli](https://docs.harmony.one/home/general/wallets/harmony-cli) or metamask and transfer funds from the following account.
+
+
+```
+On localnet, by default, 0xA5241513DA9F4463F1d4874b548dFBAC29D91f34 has funds, as defined in core/genesis.go. The private key for this address is 1f84c95ac16e6a50f08d44c7bde7aff8742212fda6e4321fde48bf83bef266dc
+```
+
+For testing purposes we used account `0x8875fc2A47E35ACD1784bB5f58f563dFE86A8451` and funded it with 1000 ONE on localnet and 1 ETH on Ropsten, but you can use any acccount you like as long as you know the private key or mnemonic.
+
+## Creating the DAG Merkle Tree
+
+DAG genertion takes several hours to run Ganesha has a machine to do this and shares the latest DAG info from Ropsten using [google drive](https://drive.google.com/file/d/1FqLCO5oc1xDYNMuub7xAqnb6kfohdf-U/view?usp=sharing). The epoch logic is the block Number divided by 30,000 so current Ropsten EPOCH is block 12280236 / 30000 = 409 which is the DAG info shared above.
+
+To run this command from the CLI you would
+```
+cd cli
+node index.js dagProve generate 409
+```
+which calculate merkle root for epochs from [start, start+n)
+
+**Note: You do not need to generate the DAG for the Harmony Chain, just Ethereum (Ropsten)**
+
+If to find the latest epoch on Harmony you can look at [explorer](https://staking.harmony.one/validators/testnet/one1fxazl9qk7c30mk3lp7tun4yepptwvfq9ss28u7) for a validater and see the latest epoch on the EXPECTED RETURN HISTORY GRAPH. For example in Harmony Testnet on May 18th 2022, the latest epoch was 75762 so to generate the merkle root would need to run the following `node index.js dagProve generate 75762`.
+
+ *Note: Moving forward we need to update DAG information for every new EPOCH on Ethereum(Ropsten)*
+
+ **Original Documentation**
+
+Original documenation is found [here](https://github.com/johnwhitton/horizon/tree/main/cli#dag-merkel-tree-cli) and uses the cli
+#### DAG Merkel Tree CLI
+1. `node index.js dagProve generate` which calculate merkle root for epochs from [start, start+n)
+```
+node index.js dagProve generate 377
+```
+2. `node index.js dagProve blockProof` which accepts block number to calculate all necessary information in order to prove the block
+```
+node index.js dagProve blockProof --block 11266784 --url https://ropsten.infura.io/v3/<project-id>
+```
+
+**Questions**
+
+1. When do you need to run `node index.js dagProve blockProof`?
+2. Do you need to generate a new DAG information on each EPOC change?
+3. Where is this currently hosted?
+4. Where do we want to deploy this long term?
+
+## Deploying Smart Contracts
+
+Following is an overview of the contracts used in the bridge and which chain they should be deployed on. *Note: her we only focus on the contracts deployed the imported contracts are not covered*
+
+**Ethereum (Kovan)**
+* HarmonyLightClient.sol : stores all blockheaders which are used for verification.
```

Comment:
Harmony block headers - I think it would be informative to document the differences compared to Ethereum block headers, and how the light clients act differently (frequency and nature of updates, cost considerations, etc.)